integration/shareddata/scalars.go

```diff
@@ -339,11 +361,30 @@ var Int64s = &Values[string]{
 		"int64-zero": int64(0),
 		"int64-max":  int64(math.MaxInt64),
 		"int64-min":  int64(math.MinInt64),
-		"int64-big":  int64Big,
 		// "int64-null": nil, TODO: https://github.com/FerretDB/FerretDB/issues/1821
 		"int64-1": int64(1099511628000),
 		"int64-2": int64(281474976700000),
 		"int64-3": int64(72057594040000000),
+
+		// long big values ~1<<61
+		"int64-big":       longBig,
+		"int64-big-plus":  longBig + 1,
+		"int64-big-minus": longBig - 1,
+
+		// long representation of double max precision ~1<<53
+		"int64-prec-max":       int64(doubleMaxPrec),
+		"int64-prec-max-plus":  int64(doubleMaxPrec) + 1,
+		"int64-prec-max-minus": int64(doubleMaxPrec) - 1,
+
+		// negative long big values ~ -(1<<61)
+		"int64-neg-big":       -longBig,
+		"int64-neg-big-plus":  -(longBig + 1),
+		"int64-neg-big-minus": -(longBig - 1),
+
+		// long representation of double max precision ~ (1<<53)
```

Comment:
```suggestion
		// long representation of double max precision ~ -(1<<53-1)
```
I'm pretty sure it's just a typo